webapp/components/wpt-results.html

```diff
@@ -406,20 +412,12 @@ <h3>
         return url;
       }
 
-      observeLoadingCount(newValue, oldValue) {
-        if (newValue === oldValue) {
-          return;
-        }
-        const elem = this.shadowRoot.querySelector('paper-spinner-lite');
-        if (!elem) {
-          return;
-        }
+      computeIsLoading(loadingCount) {
+        return loadingCount;
```

Comment:
nit: Explicit coercing to Boolean a bit clearer: `!!loadingCount`.

Reply:
Done.